src/bantz/google/consent_wizard.py

```
@@ -0,0 +1,566 @@
+    def _check_client_secret(self) -> bool:
+        """Verify client_secret.json exists."""
+        from bantz.google.auth import get_google_auth_config
+
+        cfg = get_google_auth_config()
+        if cfg.client_secret_path.exists():
+            return True
+
+        print(f"\n{_C.RED}  ✗ Google OAuth client_secret.json bulunamadı.{_C.RESET}")
+        print(f"  {_C.DIM}Beklenen konum: {cfg.client_secret_path}{_C.RESET}")
+        print()
+        print(f"  {_C.YELLOW}Nasıl alınır:{_C.RESET}")
```

Ruff (0.15.0)

[warning] String contains ambiguous `ı` (LATIN SMALL LETTER DOTLESS I). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)